github-guidelines.md

```diff
@@ -0,0 +1,91 @@
+## GIT AND GITHUB
+***
+Before continuing we want to clarify the difference between Git and Github. Git is a version control system(VCS) which is a tool to manage the history of our Source Code. GitHub is a hosting service for Git projects.
+
+We assume you have created an account on Github and installed Git on your System.
+
+Now tell Git your name and E-mail (used on Github) address.
+
+``` $ git config --global user.name "YOUR NAME" ```
+```$ git config --global user.email "YOUR EMAIL ADDRESS"``` <br>
```

Comment:
This is not a way to write single line code. Either enclose each command inside a single backtick or enclose both commands inside three backticks like this:

\`\`\`
$ git config --global user.name "YOUR NAME"
$ git config --global user.email "YOUR EMAIL ADDRESS"
\`\`\`

Also, don't use `<br>`, instead just add two whitespaces at the end of the line.

Reply:
I will do that